lib/ig.js

```diff
@@ -773,12 +773,24 @@ ${match[1]}
         // See: https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/DSTU.20ig.2Exml.20package.2Eresource.2Epurpose.20for.20SearchParameters/near/168272354
         pushXmlResource(`${exResource.resourceType}/${exResource.id}`, name, 'example');
 
-        htmlRows.push(
-          `<tr>
-            <td><a href="${exResource.resourceType}-${exResource.id}.html">${name}</a></td>
-            <td>${markdownifiedText(exResource.description)}</td>
-          </tr>
-        `);
+        if (exResource.resourceType === 'SearchParameter') {
+          const baseTypes = exResource.base.join(", ");
```

Comment:
In DSTU2, `base` is actually `1..1` -- so it's not an array.  Since we support DSTU2, STU3, and R4, we need to handle each possible version of `SearchParameter` -- so you'll want to check if this is an array before `join`ing it.

See: http://hl7.org/fhir/DSTU2/searchparameter.html

Reply:
Fixed here https://github.com/standardhealth/shr-fhir-export/pull/162/commits/eda87ebd35bafc5ec3a3dfebe766d2cfd1ec77d9.